types/field_type.go

```diff
@@ -29,6 +29,23 @@ const (
 	UnspecifiedLength int = -1
 )
 
+var (
+	// Constant MySQL Tiny
+	ConstMySQLTiny = NewFieldType(mysql.TypeTiny)
```

Comment:
Actually, these variables are not constants. We may meet problem when they are modified somewhere.